spring-boot-kotlin-io/src/main/kotlin/com/baeldung/nonblockingcoroutines/handlers/ProductsHandler.kt

```diff
@@ -0,0 +1,48 @@
+package com.baeldung.nonblockingcoroutines.handlers
+
+import com.baeldung.nonblockingcoroutines.controller.ProductStockView
+import com.baeldung.nonblockingcoroutines.model.Product
+import com.baeldung.nonblockingcoroutines.repository.ProductRepositoryCoroutines
+import kotlinx.coroutines.Deferred
+import kotlinx.coroutines.FlowPreview
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.async
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.MediaType
+import org.springframework.stereotype.Component
+import org.springframework.web.reactive.function.client.WebClient
+import org.springframework.web.reactive.function.client.awaitBody
+import org.springframework.web.reactive.function.client.awaitExchange
+import org.springframework.web.reactive.function.server.ServerRequest
+import org.springframework.web.reactive.function.server.ServerResponse
+import org.springframework.web.reactive.function.server.bodyAndAwait
+import org.springframework.web.reactive.function.server.json
+
+@Component
+class ProductsHandler(
+        @Autowired var webClient: WebClient,
+        @Autowired var productRepository: ProductRepositoryCoroutines) {
```

Comment:
Indentation in this file seems out of alignment. Would `@Autowired` be on the same line as the `var` definition?

Reply:
Hi @ashleyfrieze , as far as I know, this should be formatted in that way. This a primary constructor of the class and arguments are separated by a comma. Here are some official formatting conventions https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting